packages/panels/src/components/nav-bar/NavBarHeading.tsx

```diff
@@ -4,5 +4,5 @@ import { Heading, HeadingProps } from "@stenajs-webui/core";
 interface NavBarHeadingProps extends HeadingProps {}
 
 export const NavBarHeading: React.FC<NavBarHeadingProps> = (headingProps) => {
-  return <Heading whiteSpace={"nowrap"} color={"#fff"} {...headingProps} />;
+  return <Heading whiteSpace={"nowrap"} variant={"h4"} {...headingProps} />;
```

Comment:
Should we use `as={"h1"}` here? It is most probably the "main" heading of the app?

Reply:
Yes, good idea!